src/components/ExplorerIntroNational.js

```diff
@@ -24,7 +24,14 @@ const ExplorerIntroNational = ({ crime, ucr, until }) => {
               reports sent to the FBI.
             </p>
             <p className="serif">
-              In {until}, the FBI estimated crime statistics for the nation
+              In
+              {' '}
+              {until}
+              , the FBI
+              {' '}
+              {estimatedTerm}
```

Comment:
Next, we want to use that `estimatedTerm` variable in our text. In our current system, when you want to use a variable in some HTML you have to surround it with curly braces.